# HG changeset patch
# User Francois Bissey <francois.bissey@canterbury.ac.nz>
# Date 1331088355 -46800
# Node ID 346839e5825642add6235a07803e0e3c3b019088
# Parent  68e89260148df131fec1708338ceba3ea964b2bb
correct the mess done by 4539 to module_list.py

diff --git a/module_list.py b/module_list.py
--- a/module_list.py
+++ b/module_list.py
@@ -1545,8 +1545,8 @@
               sources = ['sage/rings/polynomial/plural.pyx'],
               libraries = ['m', 'readline', 'singular', 'givaro', 'gmpxx', 'gmp'],
               language="c++",
-              include_dirs = [SAGE_ROOT +'/local/include/singular'],
-              depends = [SAGE_ROOT + "/local/include/libsingular.h"]),
+              include_dirs = [SAGE_INC + 'singular'],
+              depends = singular_depends),
 
     Extension('sage.rings.polynomial.multi_polynomial_libsingular',
               sources = ['sage/rings/polynomial/multi_polynomial_libsingular.pyx'],
